recipes/emeraldbgc/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "emeraldbgc" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "ad6d2c6ccdca4db504b66c7ac0094acb3bdcf01ae366fce39754cfb7cca08e0e" %}
+
+package:
+  
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/Finn-Lab/emeraldBGC/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - emeraldbgc = emeraldbgc._cli:main
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - perl >=5
+    - biopython
+    - numpy >=1.16,<1.20
+    - numpy <1.20
+    - openjdk >=11.0
+    - tensorflow =2.4.1
```

Comment:
Any chance you can be a bit more flexible with the pinning? Do you really need those exact versions for TF and sklearn?

Reply:
I loosened the versions a bit. Bigger changes modify the results

Reply:
Okay :+1: